src/libs/actions/Task.ts

```diff
@@ -15,23 +15,41 @@ import * as UserUtils from '@libs/UserUtils';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import type * as OnyxTypes from '@src/types/onyx';
+import type {Icon} from '@src/types/onyx/OnyxCommon';
 import {isNotEmptyObject} from '@src/types/utils/EmptyObject';
 import * as Report from './Report';
 
-let currentUserEmail;
-let currentUserAccountID;
+type OptimisticReport = Pick<OnyxTypes.Report, 'reportName' | 'managerID' | 'participantAccountIDs' | 'notificationPreference' | 'pendingFields' | 'visibleChatMemberAccountIDs'>;
+type Assignee = {
+    icons: Icon[];
+    displayName: string;
+    subtitle: string;
+};
+type ShareDestination = {
+    icons: Icon[];
+    displayName: string;
+    subtitle: string;
+    displayNamesWithTooltips: ReportUtils.DisplayNameWithTooltips;
+    shouldUseFullTitleToDisplay: boolean;
+};
+type PolicyValue = ValueOf<typeof CONST.POLICY.ROLE>;
+
+let currentUserEmail = '';
+let currentUserAccountID = -1;
+
 Onyx.connect({
     key: ONYXKEYS.SESSION,
-    callback: (val) => {
-        currentUserEmail = lodashGet(val, 'email', '');
-        currentUserAccountID = lodashGet(val, 'accountID', 0);
+    callback: (value) => {
+        currentUserEmail = value?.email ?? '';
+        currentUserAccountID = value?.accountID ?? -1;
```

Comment:
Is there any specific reason for changing the default value for `currentUserAccountID` from `0` to `-1`? If not, it's best to leave it as `0`.

Reply:
this is what we agreed earlier, also the same is used in other places. reason `0` is possible value for `accountID`